stream_alert_cli.py

```diff
@@ -123,6 +123,11 @@ def build_parser():
         help='Names of rules to test, separated by spaces'
     )
 
+    lambda_parser.add_argument(
+        '-l', '--live',
```

Comment:
Yeah let's make it two arguments, or a separate command altogether (`live-test` maybe?)